docker/parse-html/app.rb

```diff
@@ -124,3 +131,29 @@ def extract_abbreviations(html_doc)
 
   abbreviations
 end
+
+def extract_image(html_doc)
+  images = []
+
+  html_doc.css("img").each do |images|
+    images.push({
+      "title" => images.attribute("title"), # Expansion
+      "text" => images.text, # Images
+    })
+  end
+
+  images
+end
+
+def extract_table(html_doc)
+  tables = []
+
+  html_doc.css("table").each do |tables|
+    tables.push({
+      "title" => tables.attribute("title"), # Expansion
+      "text" => tables.text, # Tables
+    })
+  end
```

Comment:
HTML tables don't seem to have any attributes, so I'm not sure what to put into this object. Maybe the HTML of the whole table, as a text string?

https://www.w3schools.com/tags/tag_table.asp